services/galley/test/integration/API/Teams.hs

```diff
@@ -187,7 +188,6 @@ testCreateOne2OneWithMembers g b c a = do
         Util.addTeamMemberInternal g tid mem1
         checkTeamMemberJoin tid (mem1^.userId) wsMem1
         assertQueue "team member join" a $ tUpdate 2 [owner]
-        WS.assertNoEvent timeout [wsMem1]
```

Comment:
Hmmm, why was this removed?

Reply:
sometimes, a `user.activated` events is getting in the way of this.  given that we create two users, we assumed that this is correct behavior and we would have to sort out a race condition.

alternatively, we could pick an event and make sure it is one of (1) not there and (2) the one we're seeing every now and then.

Reply:
👍